src/components/templates/PartyMap/components/RoomModalOngoingEvent/RoomModalOngoingEvent.tsx

```diff
@@ -1,45 +1,28 @@
-import React, { useCallback } from "react";
```

Comment:
[note] All of this stuff removed from this file was moved into `src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx`